samcli/lib/providers/sam_base_provider.py

```diff
@@ -27,6 +27,13 @@ class SamBaseProvider:
     CLOUDFORMATION_STACK = AWS_CLOUDFORMATION_STACK
     DEFAULT_CODEURI = "."
 
+    CODE_PROPERTY_KEYS = {
```

Comment:
`sam package` already has a bunch of these constants for package-able code paths, would it make sense to refactor that, so thats its usable across the code base?

Reply:
good idea

Reply:
I have another PR ready for this purpose